packages/p2p/src/components/i18next/index.ts

```diff
@@ -13,6 +13,8 @@ import fr from 'Translations/fr.json';
 import id from 'Translations/id.json';
 import it from 'Translations/it.json';
 import ko from 'Translations/ko.json';
+//TODO: remove comment once mn.json for p2p is implemented
+// import mn from 'Translations/mn.json';
```

Comment:
When are we getting that?

Reply:
I've checked with Putri on this, she said that the translation team now lack of resources on proof of read for Mongolian. She will try to check with ChatGPT and push it 🙏